packages/google-compute-engine/packaging/google-compute-engine.spec

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Name: google-compute-engine
+Version: %{_version}
+Release: 1.el7
+Summary: Google Compute Engine guest environment.
+License: ASL 2.0
+Url: https://github.com/GoogleCloudPlatform/compute-image-packages
+Source0: %{name}_%{version}.orig.tar.gz
+Requires: curl
+Requires: google-compute-engine-oslogin
+Requires: python-google-compute-engine = %{version}
+Requires: rsyslog
+
+BuildArch: noarch
+BuildRequires: systemd
+
+# Allow other files in the source that don't end up in the package.
+%define _unpackaged_files_terminate_build 0
+
+%description
+This package contains scripts, configuration, and init files for features
+specific to the Google Compute Engine cloud environment.
+
+%prep
+%autosetup
+
+%install
+cp -a src/{etc,usr} %{buildroot}
+install -d %{buildroot}/{%{_unitdir},%{_presetdir},%{_udevrulesdir}}
+cp -a src/lib/systemd/system/* %{buildroot}/%{_unitdir}
+cp -a src/lib/systemd/system-preset/* %{buildroot}/%{_presetdir}
+cp -a src/lib/udev/rules.d/* %{buildroot}/%{_udevrulesdir}
+
+%files
+%defattr(0644,root,root,0755)
+%attr(0755,-,-) %{_bindir}/*
+%attr(0755,-,-) /etc/dhcp/dhclient.d/google_hostname.sh
+%{_udevrulesdir}/*
+%{_unitdir}/*
+%{_presetdir}/*
+%config /etc/modprobe.d/*
+%config /etc/rsyslog.d/*
+%config /etc/sysctl.d/*
+#/usr/lib/systemd/journald.conf.d/*
```

Comment:
Why is this commented out? Can we remove it?

Reply:
oh, i dont think `debian/install` supports comments.. removed instead, will add comment to the file itself